practice5/src/test/java/com/academiaDigital/softwareArchitecture/practice5/products/SpaceStationTest.java

```diff
@@ -0,0 +1,53 @@
+package com.academiaDigital.softwareArchitecture.practice5.products;
+
+import com.academiaDigital.softwareArchitecture.practice5.parts.ControlPanel;
+import com.academiaDigital.softwareArchitecture.practice5.parts.Engine;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class SpaceStationTest {
+    //Arrange
+    public Engine engine = new Engine(10,"Keroseno");
+    public SpaceStation spaceStation = new SpaceStation(10,engine,50,2);
+
+    @Test
+    public void getNumberOfEngines_should_return_number_of_Engines() {

```

Comment:
expectations can come as a parameter of the assert: `assertEquals("should return number of engines", 10, numberofEngines)`